scripts/update-lit.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+CALLER_DIR=$(pwd)
```

Comment:
Nice implementation, just a thought, we could also allow the directory context to be passed as an argument to the script for flexibility.

Reply:
yea sure, can do that update as well.
initial idea was to ease down the flow but yea this would improve the flexibility as well